submissions/AnzhelikaKh/popup_task/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>PopUp</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="logo"><a href="./index.html">
+                <img class = "logo__img" src="./img/Logo1.jpg" alt="Logo"></a>
+            </div>
+            <nav class="menu">
+                <ul class="menu__list">
+                    <li><a href="#" class="menu__item">Mail</a></li>
+                    <li><a href="#" class="menu__item">Images</a></li>
+                    <li class="menu__item">
+                        <input type="checkbox" class="checkbox__main" id="checkbox__main">
+                        <label for="checkbox__main" class="main__label">
+                            <img src="./img/popup-button.png" alt="popup-button">
+                        </label>
+                        <ul class="popup__list">
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img class="item__view__img" src="./img/apps.png" alt="Icon Apps item">
```

Comment:
For decorative images implemented with `img` tag, `alt` attribute should be an empty value. The icon is a decorative image.

Reply:
I'm sorry, I already fixed it 👍